src/jumio-api/jumio-api.service.ts

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { BadRequestException, Injectable } from '@nestjs/common';
+import axios, { AxiosRequestConfig } from 'axios';
+import { ApiConfigService } from '../api-config/api-config.service';
+import { KycDetails } from '../jumio-kyc/kyc.service';
+
+@Injectable()
+export class JumioApiService {
+  constructor(private readonly config: ApiConfigService) {}
+
+  async createAccountAndTransaction(
+    userId: number,
+    jumioAccountId: string | null,
+  ): Promise<KycDetails> {
+    const jumioUrl = this.config.get<string>('JUMIO_ACCOUNT_URL');
```

Comment:
we should store the root here in the config instead of the account and build the account url. If we want to hit more apis later, we'll have to add more urls.